src/pages/public/Companion/components/CompanionLayout.js

```diff
@@ -340,142 +396,170 @@ const CompanionLayout = (params) => {
+                <div>
+                  <FadeInWhenVisible style={{
+                    ...styles.column,
+                    position: "fixed",
+                    top: "0",
+                    left: "0",
+                    right: "0",
+                    width: "100%",
+                    background: "#060818",
+                    zIndex: "9999",
+                    paddingBottom: "0.75rem"
+                  }}>
+                    <img src={Logo} alt={`${title} Logo`} style={renderMobileOnly ? styles.mobileHomeLogo : styles.homeLogo} />
+                    <nav role="navigation" style={{
+                      ...styles.nav,
+                      ...(renderMobileOnly && {
+                        width: "100%"
+                      })
+                    }}>
+                      {welcomeData && !disableWelcomeHeader && <a href="#Welcome" className={classes.link} style={{
+                        fontSize: constantStyles.fontSize
+                      }}>Welcome</a>}
+                      {scheduleData?.length && <a href="#Schedule" className={classes.link} style={{
+                        fontSize: constantStyles.fontSize
+                      }}>Schedule</a>}
+                      {headers.map((header, i) => {
+                        const lastTabItem = i === headers.length - 1 ? {
+                          borderRight: "none"
+                        } : {
+                        };
+                        if (header.id) {
+                          return (<a href={`#${header.id}`} key={i} className={classes.link} style={{
+                            fontSize: constantStyles.fontSize,
+                            ...lastTabItem
+                          }}>{header.text}</a>);
+                        } else if (header.route) {
+                          return <Link to={header.route} className={classes.link} key={i} style={{
+                            fontSize: constantStyles.fontSize,
+                            ...lastTabItem
+                          }}>{header.text}</Link>;
+                        }
+                      })}
+                    </nav>
+                  </FadeInWhenVisible>
+                </div>
+                <div style={{
+                  background: "transparent",
+                  height: "110px"
+                }}> </div>
+                {event && event.isCompleted && event.feedback ?
+                  <FadeInWhenVisible>
+                    <FeedbackForm feedbackLink={event.feedback} renderMobileOnly={renderMobileOnly} styles={styles} />
+                  </FadeInWhenVisible> : <></>}
+                <FadeInWhenVisible id="welcome" style={styles.column}>
+                  <h1 id="Welcome" style={renderMobileOnly ? styles.mobileTitle : styles.title}>Hello, {userRegistration.fname}!</h1>
+                  {event.isCompleted ? <div style={{
```

Comment:
ran into an error reading .isCompleted on a null object.

change to event && event.isCompleted